### PR TITLE
Automate resolved issues for 5.0.1

### PR DESCRIPTION
**Pull request by ilyeskallala**

### CODE REVIEW

File: forum/src/main/java/org/exoplatform/platform/qa/ui/forum/pageobject/ForumTopicManagement.java [hunk 1]
```diff
@@ -511,8 +511,10 @@ public void startTopic(String title, String message, String pathFile, String fil
     switchTo().defaultContent();
     info("click on Attached file button");
     $(ELEMENT_START_TOPIC_ATTACH_FILE).click();
-    File file = $(By.className("file")).uploadFromClasspath("data/forum/topic_attachment.txt");
-    assertTrue(file.exists());
+    for (int i=0;i<=fileName.length-1;i++){
+    $(By.className("file")).uploadFromClasspath(fileName[i]);
+
```
Comment (thomasdelhomenie) [inline forum/src/main/java/org/exoplatform/platform/qa/ui/forum/pageobject/ForumTopicManagement.java]: Indentation is not good + useless blank line.

File: platform/src/test/java/org/exoplatform/platform/qa/ui/platform/social/SOCSpaceManagementTestIT.java [hunk 2]
```diff
@@ -570,4 +580,29 @@ public void test13_CheckWhenUserIsNOTMemberOfSpace() {
     addUsers.deleteUser(username1);
     addUsers.deleteUser(username2);
   }
+
+  @BugInPLF("SOC-6043")
```
Comment (thomasdelhomenie) [inline platform/src/test/java/org/exoplatform/platform/qa/ui/platform/social/SOCSpaceManagementTestIT.java]: SOC-6043 has been fixed.

File: forum/src/main/java/org/exoplatform/platform/qa/ui/forum/pageobject/ForumTopicManagement.java [hunk 3]
```diff
@@ -493,33 +489,28 @@ public void replyTopic(String newTitle, String newMessg, String pathFile, String
...
     switchTo().defaultContent();
     info("click on Attached file button");
-    if (fileName!="") {
-      $(ELEMENT_START_TOPIC_ATTACH_FILE).click();
-      File file = $(By.className("file")).uploadFromClasspath(fileName);
-      assertTrue(file.exists());
-      $(ELEMENT_SAVE_BTN).pressEnter();
+    $(ELEMENT_START_TOPIC_ATTACH_FILE).click();
+    for (int i=0;i<=fileName.length-1;i++){
+    $(By.className("file")).uploadFromClasspath(fileName[i]);
```
Comment (thomasdelhomenie) [inline forum/src/main/java/org/exoplatform/platform/qa/ui/forum/pageobject/ForumTopicManagement.java]: Indentation

File: platform/src/test/java/org/exoplatform/platform/qa/ui/platform/social/SOCSpaceManagementTestIT.java (hunk 2 above)
Comment (aboughdiri) [reply platform/src/test/java/org/exoplatform/platform/qa/ui/platform/social/SOCSpaceManagementTestIT.java]: Done

File: forum/src/main/java/org/exoplatform/platform/qa/ui/forum/pageobject/ForumTopicManagement.java (hunk 3 above)
Comment (aboughdiri) [reply forum/src/main/java/org/exoplatform/platform/qa/ui/forum/pageobject/ForumTopicManagement.java]: Done

**Pull request merged**